lib/migration.js

```diff
@@ -333,7 +333,7 @@ function mixinMigration(MsSQL) {
         } else if (idProp.type === String) {
           if (idProp.generated !== false) {
             sql.push(self.columnEscaped(model, modelPKID) +
-              ' [uniqueidentifier] DEFAULT newid() NOT NULL');
+              ' [varchar](128) DEFAULT newid() NOT NULL');
```

Comment:
I don't think this the correct fix for a few reasons:

1. The proposed change is breaking compatibility
2. Please note `newid()` returns `uniqueidentifier`
3. The test probably assumes id is always a string 

We probably need to fix the test case instead.